src/constructs/lambda/lambda.ts

```diff
@@ -92,9 +96,12 @@ function defaultMemorySize(runtime: Runtime, memorySize?: number): number {
 export class GuLambdaFunction extends Function {
   public readonly app: string;
   public readonly fileName: string;
+  public readonly bucketNamePath: string | undefined;
+  public readonly managedDeployment: boolean;
+  public readonly withoutFilePrefix: boolean;
 
   constructor(scope: GuStack, id: string, props: GuFunctionProps) {
-    const { app, fileName, runtime, memorySize, timeout, bucketNamePath, withoutFilePrefix } = props;
+    const { app, fileName, runtime, memorySize, timeout, bucketNamePath, withoutFilePrefix, managedDeployment } = props;
```

Comment:
I think destructuring with default values would be simpler here, making the value more immediately obvious, and removes the need to nullish coalesce:

```ts
const {
  app,
  fileName,
  runtime,
  memorySize,
  timeout,
  buckNamePath,
  withoutFilePrefix = false,
  withoutFilePrefix = false
} = props;
```